client/app/services/consoles.service.js

```diff
@@ -47,8 +47,15 @@ export function ConsolesFactory ($window, CollectionsApi, $timeout, $location, E
     }
   }
 
-  function consoleError (error) {
-    EventNotifications.error(__('There was an error opening the console. ' + error.message))
+  function consoleError (response) {
+    let message = __('undefined')
+    if (response.data) {
+      message = response.data.error.message
+    } else if (response.message) {
```

Comment:
When would this case happen (the response.message)? I thought error messages were always of the `response.data.error.message` format?

Reply:
I dug through our test cases and it seems that in the use case that the response was a 200 and status was != success .  I am assuming the test cases were accurate as I didn't have a way to forcibly replicate this http response scenario.  This change I made supports that scenario and seems harmless enough in the case the scenario doesn't happen.

Reply:
Hokay, yeah no harm, merging.